netlify.toml

```diff
@@ -4,3 +4,8 @@ command = "npm run build:preview"
 
 [context.production]
 command = "npm run build:production"
+
+[[redirects]]
+from = "https://docs.opentelemetry.io/*"
+to = "https://opentelemetry.io/docs/:splat"
+status = 200
```

Comment:
I don't think that you want a status field here. Using the default 301 seems appropriate to me.

Nit: ensure that there's a newline at the end of the file.

Reply:
Oh! I see that using 200 keeps the URL as is. Interesting.